components/01-globals/icons/icons.config.js

```diff
@@ -5,6 +5,8 @@ module.exports = {
       'angle-down',
       'arrow-circle-up',
       'arrow-left',
+      'arrow-small-down',
+      'arrow-small-up',
```

Comment:
i think their namings are wrong b/c it's not small icons. It can be XXXXpx width and height

Reply:
don't ignore this 😄

Reply:
still this names exists

Reply:
so arrow-type2 ? :) what name do you suggest?

Reply:
you have issue with icons. I see it's similar to "angle-down" (a see a minimal difference) can you than make order with all icons in the other issue with icons ?

Reply:
so I would leave it here like it is for later reviewing icon graphics and naming

Reply:
it's already merged, so get the latest develop, next delete `arrow-small-down` b/c it's the same as 'angle-down' and renames 'arrow-small-up' to 'angle-up'.

Reply:
ok. so changed. there are not the same but I will post issue on design to replace with proper icons the arrows